src/v2/Artsy/Router/__tests__/buildServerApp.jest.tsx

```diff
@@ -226,29 +235,43 @@ describe("buildServerApp", () => {
 
   describe("concerning GraphQL errors", () => {
     const consoleError = console.error
+    jest.mock("v2/Artsy/Relay/createRelaySSREnvironment", () => ({
+      createRelaySSREnvironment: jest.fn(),
+    }))
```

Comment:
👍  I never realized that its possible to declare `jest.mock` statements outside of the top